app/js/components/demographics.jsx

```diff
@@ -198,187 +209,205 @@ export default class Demographics extends React.Component {
+                                                                this.state.deathDate :
+                                                                'Select date of death'
+                                                            }
+                                                            inputProps={{ disabled: true }}
+                                                            format={this.state.format}
+                                                            viewMode={this.state.mode}
+                                                            inputFormat={this.state.inputFormat}
+                                                            dateTime={this.state.deathDate}
+                                                            onChange={this.handleTimeChange('deathDate')}
+                                                        />
+                                                    </div>
+                                                </div>
+                                                <label className='control-label col-sm-4'> Estimated: </label>
+                                                <div className='control-label col-sm-8'>
+                                                    <input className='control-label col-sm-8'
+                                                        name='estimated'
+                                                        type='checkbox'
+                                                        value={this.state.deathDateEstimated}
+                                                    />
+                                                </div>
+                                            </div>
+                                            {
+                                                this.state.deathDateError &&
+                                                <div className='form-group has-error err-msg'>
+                                                    <div className='err-msg'>
+                                                        {this.state.deathDateError}
+                                                    </div>
+                                                </div>
+                                            }
+                                            <div className='form-group'>
+                                                <div className={this.state.causeOfDeathError ?
+                                                    'has-error' : ''
+                                                }>
+                                                    <label className='control-label col-sm-4'>
+                                                        Cause of death:
+                      </label>
+                                                    <div className='control-label col-sm-8'>
+                                                        <select className='form-control'
+                                                            onChange={this.handleChange}
```

Comment:
This closing label tag is out of indent

Reply:
@jacksono It is correctly indented. Just happens to be on its own line